data_mapper_set_up.rb

```diff
@@ -0,0 +1,10 @@
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, "postgres://localhost/chittter_#{env}")
+
+require './lib/user'
+require './lib/peep'
+require './lib/reply'
+
+DataMapper.finalize
+DataMapper.auto_upgrade!
```

Comment:
when you'll do the migrate for the test database you can do this: rake auto_migrate 'RACK_ENV'=test